src/Holycode.Configuration.CLI/Holycode.Configuration.CLI.csproj

```diff
@@ -4,10 +4,14 @@
     <VersionPrefix>3.0.3</VersionPrefix>
     <VersionSuffix>master000-5b427</VersionSuffix>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
z tego co widzę tutaj
https://docs.microsoft.com/pl-pl/dotnet/standard/net-standard?tabs=net-standard-2-0
można by się pokusić o używanie najnowszego net6.0

Reply:
dotyczy wszystkich projektów

Reply:
tutaj zmieniłem, a jakie dodatkowe projekty masz na myśli? w tym PR to chyba jedyny projekt który się kompiluje do pliku wykonywalnego

Reply:
Ale zmieniłeś na net core 3, czemu nie net5 lub net6 (najlepiej)?
Wydawało mi się, że paczka może targetować net6 i być zgodna z netstandard2. Jeżeli tak nie jest to nie było tematu.